### ISSUE TITLE
error: version 0.25.0 ValueError: No JSON object could be decoded

### ISSUE THREAD

**Issue by eric20141111** [jaysonsantos/python-binary-memcached#72]
I'm using python-binary-memcached for my app api server data cache.
I installed python-binary-memcached==0.24.6 for the first time.
when I update to 0.25.0, I got an error, like this:
```python
2017-01-06 10:19:29,114 - ERROR - app.py - log_exception - 1560 - Exception on /index_collections [GET]
Traceback (most recent call last):
  File "/home/jim/.virtualenvs/wewowenv/local/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/jim/.virtualenvs/wewowenv/local/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/jim/.virtualenvs/wewowenv/local/lib/python2.7/site-packages/flask_restful/__init__.py", line 477, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/jim/.virtualenvs/wewowenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/jim/.virtualenvs/wewowenv/local/lib/python2.7/site-packages/flask_restful/__init__.py", line 587, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/jim/www/wewowapi/app.py", line 3471, in get
    data = client.get(key)
  File "/home/jim/.virtualenvs/wewowenv/local/lib/python2.7/site-packages/bmemcached/client.py", line 105, in get
    value, cas = server.get(key)
  File "/home/jim/.virtualenvs/wewowenv/local/lib/python2.7/site-packages/bmemcached/protocol.py", line 428, in get
    return self.deserialize(value, flags), cas
  File "/home/jim/.virtualenvs/wewowenv/local/lib/python2.7/site-packages/bmemcached/protocol.py", line 372, in deserialize
    return loads(value)
  File "/home/jim/.virtualenvs/wewowenv/local/lib/python2.7/site-packages/bmemcached/protocol.py", line 390, in json_loads
    return json.loads(value.decode('utf8'))
  File "/usr/lib/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 384, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded

```
then I reinstall 0.24.6, the error gone.
So, is this a 0.25.0 bug?

**Comment by jaysonsantos** [comment]
Hey @eric20141111 this is a known issue. @hathawsh will you be able to tackle this?

**Comment by jaysonsantos** [comment]
This was fixed in version 0.16
